### ISSUE TITLE
Proposal for DIF kumu account

### ISSUE THREAD

**Issue by manicprogrammer** [decentralized-identity/vc-spec-map#15]
spawning from #11 propose to DIF Interop Group that: 

- if DIF does not have an account on kumu that represents the organization that one be created under the control of admins much like the github organizational account
- that once this is done or if already done a workspace for specification mapping be created with the maintainers of this repo as contributors to that workspace

This allows for an organizational account thus not requiring DIF artifacts and product for kumu to live in personal account workspaces.
